### PR TITLE
MONIT-16779: Added setting of the exact number of traces to generate.

### PR DESCRIPTION
**Pull request by dbaghdasarya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ The traces generation parameters could be simply provided via command line optio
 - `--errorRate=10` - Percentage of erroneous traces (0-100). Default: 0
 - `--rate=50` - Rate at which the spans will be ingested (integer number of spans per second). Default: 100
 - `--traceTypesCount=5` - Number of traces types for auto-generation. Default: 0
+- `--totalTraceCount=16` - Number of traces for auto-generation. If this parameter greater
```
Comment (sirak92) [inline README.md]: From the description is not clear that this parameter can be used in Advanced way also. So if this parameter is set and traceTypesPatterns are defined the generated traces count will be the input value.

File: src/main/java/com/wavefront/config/GeneratorConfig.java [hunk 2]
```diff
@@ -41,6 +41,10 @@
       "auto-generation.")
   private Integer traceTypesCount = 0;
 
+  @Parameter(names = {"--totalTraceCount"}, description = "Total number of traces for " +
+      "auto-generation. This parameter disables duration")
```
Comment (sirak92) [inline src/main/java/com/wavefront/config/GeneratorConfig.java]: the same here, the description needs to be improved

File: src/main/java/com/wavefront/config/GeneratorConfig.java [hunk 3]
```diff
@@ -73,6 +77,10 @@ public void initPropertiesFromFile() throws IOException {
     // if traceTypesCount is set, it has precedence so the traces will be generated with default
     // parameters
     traceTypesCount = rootNode.path("traceTypesCount").asInt(0);
+    totalTraceCount = rootNode.path("totalTraceCount").asInt(0);
+    if (totalTraceCount < 0) {
+      totalTraceCount = 0;
+    }
```
Comment (sirak92) [inline src/main/java/com/wavefront/config/GeneratorConfig.java]: why we need to default it to 0, as later you check if totalTraceCount > 0

Comment (dbaghdasarya) [reply src/main/java/com/wavefront/config/GeneratorConfig.java]: Agree, fixed.

File: README.md (hunk 1 above)
Comment (dbaghdasarya) [reply README.md]: totalTraceCount exists in the Advanced mode description (line 130).
Do you need more details in this section?

File: src/main/java/com/wavefront/config/GeneratorConfig.java (hunk 2 above)
Comment (dbaghdasarya) [reply src/main/java/com/wavefront/config/GeneratorConfig.java]: Could you please mention what you prefer to be improved?

File: README.md (hunk 1 above)
Comment (dbaghdasarya) [reply README.md]: Removed "auto" prefix.

File: src/main/java/com/wavefront/config/GeneratorConfig.java (hunk 2 above)
Comment (dbaghdasarya) [reply src/main/java/com/wavefront/config/GeneratorConfig.java]: Removed "auto" prefix

**Pull request merged**